Pyrado/tests/algorithms/test_algorithms.py

```diff
@@ -172,6 +172,72 @@ def test_snapshots_notmeta(ex_dir, env: SimEnv, policy, algo_class, algo_hparam)
     assert isinstance(extra, dict)
 
 
+@pytest.mark.parametrize("env", ["default_qbb"], ids=["qbb"], indirect=True)
```

Comment:
Actually, this test is not necessary since the class is tested with the others. Pls delete it

Reply:
Removed in 7af1655b29